src/pkg/packager/deploy.go

```diff
@@ -368,16 +390,22 @@ func (p *Packager) deployComponent(ctx context.Context, component v1alpha1.ZarfC
 	if err != nil {
 		return nil, err
 	}
+	l.Debug("done deploying component", "name", component.Name, "duration", time.Since(start))
 	return charts, nil
 }
 
 // Move files onto the host of the machine performing the deployment.
-func (p *Packager) processComponentFiles(component v1alpha1.ZarfComponent, pkgLocation string) error {
+func (p *Packager) processComponentFiles(ctx context.Context, component v1alpha1.ZarfComponent, pkgLocation string) error {
+	l := logger.From(ctx)
+	l.Handler()
```

Comment:
Bug: Is this intentional? It doesn't look like this value is used.

Reply:
Totally missed that good catch